common/base/include/claraparabricks/genomeworks/logging/logging.hpp

```diff
@@ -94,7 +94,11 @@
 #ifdef GW_CUDA_BEFORE_9_2
 // Due to a header file incompatibility with nvcc in CUDA 9.0
 // logging through the logger class in GW is disabled for any .cu files.
+#elif __GNUC__ >= 9
 #pragma message("Logging disabled for CUDA Toolkit < 9.2")
```

Comment:
shouldn't this line should be moved above elif?

Reply:
Totally right, it should be :). Just fixed.